tests/manage/pv_services/test_resource_deletion_during_pvc_pod_creation_and_io.py

```diff
@@ -371,6 +372,14 @@ def test_resource_deletion_during_pvc_pod_creation_and_io(
             f"{final_pods_num}"
         )
 
+        # If platform is MS and resource_to_delete is osd, verify that the OSD pods are running on nodes which are
+        # part of the correct machinepool.
+        if (
+            config.ENV_DATA["platform"].lower() in constants.MANAGED_SERVICE_PLATFORMS
```

Comment:
Please use the function `is_managed_service_cluster` in the `ocs_ci/ocs/cluster.py` file.

Reply:
Done